src/Illuminate/Database/Schema/Columns/Decimal.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Illuminate\Database\Schema\Columns;
+
+/**
+ * Class Decimal
+ *
+ * @property-read int|null $total
+ * @property-read int|null $places
```

Comment:
Any reason why you have a mix of `foo|null` and `?foo`?

Reply:
As I know these php docs are part of [PhpDocumentor](http://docs.phpdoc.org/guides/types.html#primitives). It does not support nullable types. I could be mistaken.

Reply:
Because of copy/paste:) Thank you!
I deleted ?type from everywhere because it is actually not required